src/components/HeadOfTable/HeadOfTable.tsx

```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export const HeadOfTable: React.FC = () => {
+  const handleNavLinkClass = ({ isActive }: { isActive: boolean }) => {
```

Comment:
Will be better to create a separate type for { isActive: boolean }

Reply:
It is not necessary here, but what can be done for sure is to move the `handleNavLinkClass` function outside the definition of the component